docs/content/doc/usage/command-line.en-us.md

```diff
@@ -40,6 +40,7 @@ Starts the server:
 
 - Options:
     - `--port number`, `-p number`: Port number. Optional. (default: 3000). Overrides configuration file.
+    - `--install-port number`: Port number to run the install page on. Optional. (default: 3000). Overrides configuration file.
```

Comment:
I think this change could be another PR.

Reply:
It's part of the fix really. 

If you make --port apply both before the install page and after the install page then the PORT option in the install page is irrelevant and it is overwritten.

If you make --port only apply once then if there is a graceful restart after install the port that gitea runs on will change.

If you make --port only apply to the install page then that may change behaviour for other people.

In this case the only solution is provide a separate option for the install page then you can have a consistent solution.